src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,120 @@
+.catalog {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, $catalog__card-width);
```

Comment:
The variable `$catalog__card-width` is used here, but ensure that it is defined elsewhere in your SCSS files. If not, this will cause a compilation error.